package.json

```diff
@@ -17,6 +17,7 @@
     "compile:wallet": "npx etherlime compile --workingDirectory contracts/wallet --runs=999",
     "compile:test": "npx etherlime compile --workingDirectory contracts-test --runs=999",
     "compile": "npm run compile:infrastructure && npm run compile:modules && npm run compile:wallet",
+    "cc": "rm -rf build && npm run compile:lib && npm run compile && npm run compile:legacy && npm run compile:test",
```

Comment:
This will make our life so much easier :-)

We should probably add `rm -rf build-legacy` as well.